src/pages/data-globe/data-globe-component.jsx

```diff
@@ -5,28 +5,28 @@ import LandscapeViewManager from 'components/landscape-view-manager';
 import EntryBoxes from 'components/entry-boxes';
 import Sidebar from 'components/sidebar';
 import BiodiversityLayers from 'components/biodiversity-layers';
-import HumanPressureLayers from 'components/human-pressure-layers';
+import MultipleActiveLayers from 'components/multiple-active-layers';
 
 // WIDGETS
 import LocationWidget from 'components/widgets/location-widget';
 import ZoomWidget from 'components/widgets/zoom-widget';
 import ToggleUiWidget from 'components/widgets/toggle-ui-widget';
 import SearchWidget from 'components/widgets/search-widget';
 
+// styles
+import styles from './data-globe-styles.module';
+
 const { REACT_APP_DATA_GLOBE_SCENE_ID: SCENE_ID } = process.env;
 
+
 const checkboxesLandUse = [
-  { name: 'Rainfed agriculture' },
-  { name: 'Irrigated agriculture' },
-  { name: 'Urban pressures' }
+  { name: 'Rainfed agriculture', value: 'rainfed' },
+  { name: 'Irrigated agriculture', value: 'agriculture' },
+  { name: 'Urban pressures', value: 'urban' }
 ];
 
-const checkboxesFishingActivity = [
-  { name: 'Drifting longlines' },
-  { name: 'Fixed gear' },
-  { name: 'Other' },
-  { name: 'Purse seins' },
-  { name: 'Trawlers' }
+const checkboxesWDPA = [
```

Comment:
Just thinking that maybe this should be in the constants folder, as is something that we may be using on the landscape widgets.
What do you think?

Reply:
yes, definitely, I moved it 🙆